components/cam/src/physics/rrtmgp/cam_optics.F90

```diff
@@ -499,7 +499,7 @@ subroutine set_cloud_optics_sw(state, pbuf, day_indices, kdist, optics_out)
       call handle_error(optics_out%delta_scale())
 
       ! Check cloud optics_sw
-      call handle_error(optics_out%validate())
+      call handle_error(optics_out%validate(), stop_on_error=.false.)
```

Comment:
I thought the error in asymmetry parameter was popping up in the aerosol optics, not in the cloud optics. This won't catch that, but does allow our cloud optics to go out of bounds now without failing.